frontend/tests/features/home/homePage.test.ts

```diff
@@ -26,6 +34,32 @@ describe("Homepage", () => {
 
     await isGlobalLayoutWrapperScrollable(page);
   });
+  test.only("Hero numbers are rendered", async ({ page }) => {
```

Comment:
i think this should be `test` not `test.only`

Reply:
😮‍💨